index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Building Landing Page</title>
+    <meta charset="UTF-8" />
+    <<<<<<< HEAD
+    <link rel="stylesheet" href="./css/style.css" />
+    =======
+    <link rel="stylesheet" href="styles.css" />
+    >>>>>>> 09c5a842983cb9dfb9cdfe5ffabb2f75e70671b1
+    <script
+      src="https://kit.fontawesome.com/136b6c674f.js"
+      crossorigin="anonymous"
+    ></script>
+    <script src="fccp3.js"></script>
+  </head>
+  <body>
+    <header id="header">
+      <div class="logo">
+        <img
+          id="header-img"
+          src="https://freelogocreator.com/images/logo-large.png"
+        />
+      </div>
+      <nav id="nav-bar">
+        <a href="#features" class="nav-link">Features</a>
+        <a href="#video" class="nav-link">How It Works</a>
+        <a href="#pricing" class="nav-link">Pricing</a>
+      </nav>
+    </header>
+
+    <section class="container">
+      <form
+        id="form"
+        action="https://www.freecodecamp.com/email-submit"
+        method="POST"
+      >
+        <h1>Check your account</h1>
+        <input
+          type="email"
+          placeholder="Enter your email"
+          required
+          id="email"
+        />
+        <button id="submit" class="Enter-btn">Let's Go</button>
+      </form>
+
+      <section id="video">
+        <iframe
+          width="560"
+          height="315"
+          src="https://www.youtube.com/embed/2VqzrWwTucc"
+          title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        ></iframe>
+      </section>
+
+      <section id="features">
+        <div class="icons">
+          <i class="far fa-keyboard"></i>
+          <i class="fas fa-video"></i>
+          <i class="fas fa-image"></i>
+        </div>
+
+        <div class="description">
+          <div>
+            <h2>Scripts</h2>
+            <p>Research and write scripts for different topics fastly</p>
+          </div>
+          <div>
+            <h2>Video Editing</h2>
+            <p>
+              Create and edeit videos as well as translate your ideas into them
+            </p>
+          </div>
+
+          <div>
+            <h2>Photo Editors</h2>
+            <p>Edit photos in the right way using the latest media</p>
+          </div>
+        </div>
+      </section>
+    </section>
+
+    <footer id="pricing" class="footer">
+      <section class="end">
+        <div>
+          <a href="#features">Privicy</a>
+          <a href="#features"> Terms</a>
+          <a href="#features">Contact</a>
```

Comment:
- It would be better for the list of icons to use a ul tag to group them, and li per icon 😀 
using the ul and li tags in HTML can improve the organization, structure, styling, and semantics of your content, making it more accessible, user-friendly, and maintainable.